src/components/ProfitAndLossTable/ProfitAndLossTable.tsx

```diff
@@ -18,16 +21,17 @@ type Props = {
   stringOverrides?: ProfitAndLossTableStringOverrides
 }
 
-export const ProfitAndLossTable = ({
-  lockExpanded,
-  asContainer,
-  stringOverrides,
-}: Props) => {
+export const ProfitAndLossTable = ({ asContainer, stringOverrides }: Props) => {
   const {
     data: actualData,
     isLoading,
     setSidebarScope,
   } = useContext(ProfitAndLoss.Context)
+  const { isOpen, setIsOpen } = useTableExpandRow()
+
+  useEffect(() => {
+    setIsOpen(['income'])
+  }, [])
```

Comment:
Forcing another repainting and updating is not optimal. Let's change `useTableExpandRow` so it can take prop like `defaultOpen` and initiate the state right away.

Reply:
We want to expand all first-level rows, so we need to use `['income', 'cost_of_goods_sold', 'expenses']`